src/Sfx/App/Scripts/Models/DataModels/Collections.ts

```diff
@@ -320,23 +320,22 @@ module Sfx {
             this.disabledNodes = `${disabledNodes}/${disablingNodes}`;
             this.seedNodeCount = seedNodes.length;
 
-            this.checkSeedNodeCount(this.seedNodeCount);
             this.checkOneNodeScenario();
 
             this.healthySeedNodes = seedNodes.length.toString() + " (" +
                 Math.round(healthyNodes.length / seedNodes.length * 100).toString() + "%)";
         }
 
-        private checkSeedNodeCount(count: number) {
-            //if < 5 seed nodes display warning
-            //if count is 1, it is one box/test only scenario
-            if (count < 5 && count !== 1) {
+        public checkSeedNodeCount(expected: number) {
+            if (this.seedNodeCount < expected && this.seedNodeCount !== 1) {
                 this.data.warnings.addOrUpdateNotification({
-                    message: "Cluster is degraded because it does not have 5 seed nodes. See link for more details",
-                    level: StatusWarningLevel.Error,
+                    message: `This cluster is currently running on the bronze reliability tier. For production workloads a reliability level of silver or greater is recommended. 

```

Comment:
For production workloads a reliability => For production workloads, a reliability